core/templates/services/UpgradedServices.ts

```diff
@@ -1222,7 +1223,10 @@ export class UpgradedServices {
     upgradedServices['TranslationsBackendApiService'] =
       new TranslationsBackendApiService(
         upgradedServices['HttpClient']);
-
+    upgradedServices['TranslationLanguageService'] =
+      new TranslationLanguageService(
+        upgradedServices['LanguageUtilService'],
+        upgradedServices['LoggerService']);
```

Comment:
These changes are not required, can you remove them?

Reply:
If I remove this around 17 front-end tests are failing? is it fine to remove it then as well?

Reply:
Done, I have made the changes!